code/center.m

```diff
@@ -7,4 +7,4 @@
     %   Returns a new array containing the values in
     %   DATA centered around the value.
 
-    out = (data  - mean(data)) + desired;
+    out = (data  - mean(data(:))) + desired;
```

Comment:
We did agree, but that was based on my incorrect memory of the data being vectors. The data in the example csv files are matrices, so this function should work on matrices.